components/alert.css

```diff
@@ -0,0 +1,38 @@
+.alerts .alert-info,
+.alerts .alert-warn,
+.alerts .alert-error,
+.alerts .alert-success {
+  margin: 1rem 0;
```

Comment:
you can have a standard class alert to form a alert and then alternate classes alert info , warning  etc to change colors

Reply:
exactly

Reply:
Done. Thank you! @horsemaker @prathmeshjagtap